includes/ft_printf_list.sh

```diff
@@ -99,7 +99,7 @@ then
 	CHK_FT_PRINTF_LIST+=("s" "Line Feed %s|\n")
 
 	CHK_FT_PRINTF_LIST+=("s" "%10s is a string|this")
-	CHK_FT_PRINTF_LIST+=("s" "%010s is a string|this")
+	CHK_FT_PRINTF_LIST2+=("s" "%010s is a string|this")
```

Comment:
As the first commentary: 010 means "at least 10 characters to print, right-aligned, filled with '0' on left-side"